app/controllers/api/v2/access_requests_controller.rb

```diff
@@ -1,19 +1,29 @@
 module API
   module V2
     class AccessRequestsController < API::V2::ApplicationController
-      before_action :build_access_request
+      before_action :build_access_request, except: :index
+      before_action :build_access_requests, except: :approve
 
       def approve
         result = AccessRequestApprovalService.call(@access_request)
 
         render status: 200, json: { result: result }
       end
 
+      def index
```

Comment:
Jake has mentioned this but I think it was missed.

This method should be consistent with our other controllers and read like the below so you can remove the build method. We only use a `before_action` to DRY the code up when we're repeating something on multiple actions or to make it read easier.

```
    def index
        authorize AccessRequest
        @access_requests = AccessRequest.requested

        render jsonapi: @access_requests
    end
```

Reply:
Yea sorry i missed that one. Have now actioned it. Thanks Dan.